app/src/main/res/values/strings.xml

```diff
@@ -778,16 +775,16 @@
     <string name="activity_result">Activity result</string>
     <string name="result_code">Result code</string>
     <string name="action">Action</string>
-    <string name="mime_type">MIME Type</string>
+    <string name="mime_type">MIME type</string>
```

Comment:
```suggestion
    <string name="mime_type">File associations</string>
```
LXQt uses associations, so I thought it a good compromise.

Reply:
It doesn't matter what anybody uses. Android docs says that it's a **MIME type** and MIME type shall be used to avoid confusion unless they decided to change it to something else. Instead of changing it here, follow my advice and open an issue in the Google Issue tracker first.

Reply:
My point was never the technical merit of what Google calls it in the documentation or code. It appears to be "mimeType" and "MimeType" there. Does it ever appear in any Google software to describe what I think is ~"File association"?

Reply:
What the translators translate don't really matter since they're not “official”. If they don't understand something, they have to look into the English version. Latest version of the docs (yet to be released) have this statement.